### PR TITLE
Update alpha_vantage.markdown

### PR DESCRIPTION
**Pull request by cnewman402**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @cnewman402, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16448).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by cnewman402** [comment]
inverted settings for crypto corrected.

File: source/_integrations/alpha_vantage.markdown [hunk 1]
```diff
@@ -91,7 +91,7 @@ sensor:
         currency: USD
         symbol: GOOGL
     foreign_exchange:
-      - from: BTC
+      - name: BTC
```
Comment (fabaff) [inline source/_integrations/alpha_vantage.markdown:94]: `name:` can be arbitrary and [set](https://www.home-assistant.io/integrations/alpha_vantage/#name) by the user. 

Ordering of the keys is irrelevant.

**Comment by frenck** [comment]
This PR actually doesn't change a thing. Therefore closing it.

**Pull request closed without merging**